app/src/main/java/org/dhis2/ui/DataEntryBottomDialogContent.kt

```diff
@@ -0,0 +1,207 @@
+package org.dhis2.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment.Companion.CenterHorizontally
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.dhis2.R
+import org.dhis2.commons.data.FieldWithIssue
+import org.dhis2.commons.data.IssueType
+
+@Composable
+fun DataEntryBottomDialogContent(
+    isSaved: Boolean,
+    message: String,
+    fieldsWithIssues: List<FieldWithIssue>? = emptyList(),
```

Comment:
Just an obervation:
In my opinion these three properties could be a single immutable Ui model for example:
`data class DialogContentUiModel (
val message: String,
val fieldsWithIssues: List<FieldWithIssue>? = emptyList()
)`

Regarding the boolean is saved, i think it is adding presentation logic to the Composable class to selec one string or another and maybe is more complex for testing. I think that what I would do when creating the ui model, I would compute the logic and add two string as properties. So my proposed model would be:
`data class DialogContentUiModel (
val message: String,
val fieldsWithIssues: List<FieldWithIssue>? = emptyList(),
val icon: Int,
val message2: String
)`
I put message2 becuase I don't know the difference between message and message2, maybe it is title and subtitle.

I this way all logic can be model to a ViewModel or presentation component and be tested in once. Also with a single UI model we can, in the future assert our final expected result without espresso.

Please, let me know what do you think.